### ISSUE TITLE
Karabiner_AXNotifier induced Oracle SQL Developer crash

### ISSUE THREAD

**Issue by aleksandrs-ledovskis** [tekezo/Karabiner#231]
After upgrading from KeyRemap4MacBook to Karabiner, and checking "Karabiner_AXNotifier.app" option in Security & Privacy -> Accessibility preferences menu SQLDeveloper application fails to load. Disabling aforementioned checkbox solves crash issue.

Interesting fact - running `sqldeveloper` executable from within ".app" internals by command line works no matter the preference.
Interesting fact #2 - I have (deselected) "iTerm" option listed in Accessibility pane, but checking it doesn't do any harm to SQL Developer runtime

Tried both with JDK7 and 8 - no visible difference in behavior.
SQL Developer version is latest, 4.0.2.15
OS X - 10.9.4 (13E28)
Karabiner v9.99.32 and previous stable are affected
